pydatalab/src/pydatalab/apps/uvvis/__init__.py

```diff
@@ -0,0 +1,170 @@
+import os
+from pathlib import Path
+
+import bokeh.embed
+import numpy as np
+import pandas as pd
+from bokeh.models import HoverTool
+
+from pydatalab.blocks.base import DataBlock
+from pydatalab.bokeh_plots import DATALAB_BOKEH_THEME, selectable_axes_plot
+from pydatalab.file_utils import get_file_info_by_id
+from pydatalab.logger import LOGGER
+
+
+class UVVisBlock(DataBlock):
+    accepted_file_extensions = (".txt", ".TXT")
```

Comment:
```suggestion
    accepted_file_extensions = (".txt",)
```

These should already be case-insensitive (all comparisons should be done case-insensitively)

Alternatively, you might consider specialising it really to just raw8 files (would need to test that "double extensions" like this work fine, but they should)

```suggestion
    accepted_file_extensions = (".Raw8.txt", )
```